fvtest/coretest/TestTypeTraits.cpp

```diff
@@ -113,4 +113,23 @@ TEST(TestTypeTraits, IsVoid)
 	EXPECT_FALSE((IsVoid<void*>::VALUE));
 }
```

Comment:
This file requires a proper copyright header.  Can you please add one in a separate commit?